Classes/BEMSimpleLineGraphView.m

```diff
@@ -360,11 +364,11 @@ - (void)layoutTouchReport {
                 
                 NSString *prefix = @"";
                 NSString *suffix = @"";
-                if ([self.delegate respondsToSelector:@selector(popUpSuffixForlineGraph:)]) {
-                    suffix = [self.delegate popUpSuffixForlineGraph:self];
+                if ([self.delegate respondsToSelector:@selector(popUpSuffixForlineGraph:index:)]) {
+                    suffix = [self.delegate popUpSuffixForlineGraph:self index:0];
```

Comment:
The index passed here shouldn't be 0. 
Here we're trying to determine the size of the longest possible popuplabel.
Therefore, we shouldn't get the pop up suffix at index 0, but instead the longest pop up suffix.